test/session/session.test.js

```diff
@@ -0,0 +1,132 @@
+        ...("user" in responseBody
+            ? {
+                  user: new UserClass(responseBody.user),
+              }
+            : {}),
+        ...("recipeUserId" in responseBody
+            ? {
+                  recipeUserId: SuperTokens.convertToRecipeUserId(responseBody.recipeUserId),
+              }
+            : {}),
+    }};
+};
+
+describe(`sessionTests: ${printPath("[test/session/session.test.js]")}`, function () {
+    describe("Cookie checks", function () {
+        it("access and refresh tokens set correctly on new session", async function () {
+            const connectionURI = await createCoreApplication();
+            supertokens.init({
+                supertokens: {
+                    connectionURI,
+                },
+                appInfo: {
+                    apiDomain: "api.supertokens.io",
+                    appName: "SuperTokens",
+                    websiteDomain: "supertokens.io",
+                },
+                recipeList: [EmailPassword.init(), Session.init({ getTokenTransferMethod: "cookie" })],
+            });
+
+            const { response, headers } = await signUp("public", "test@example.com", "password123");
+            const epUser = response.user;
+
+            // Create a new session for the user, get headers from the response
+            const session = await Session.createNewSessionWithoutRequestResponse("public", epUser.loginMethods[0].recipeUserId);
+
+            let cookies = headers?.["set-cookie"];
+            assert(cookies, "No cookies found in response headers");
+
+            if (!Array.isArray(cookies)) {
+                cookies = [cookies];
+            }
+
+            // Parse cookies from the response. Fastapi set-cookie responses are a large string.
+            cookies = cookies
+                .flat() // Ensure we have a flat array of cookies
+                // Split cookie strings into arrays
+                .map((cookieStr) => setCookieParser.splitCookiesString(cookieStr))
+                .flat() // Since we have an array of arrays now
+                // `parse` the cookies
+                .map(setCookieParser.parseString);
+
+            const accessTokenCookie = cookies.find((info) => (info?.key ?? info?.name) == "sAccessToken");
+            const refreshTokenCookie = cookies.find((info) => (info?.key ?? info?.name) === "sRefreshToken");
+
+            // Ensure cookies are set and with GMT timezones
+            // https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Date - Date headers are always GMT
+            // https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Set-Cookie#expiresdate - Invalid cookies become session cookies
+            assert(accessTokenCookie, "Access token cookie not found");
+            assert(accessTokenCookie.expires, "Access token cookie expiry not set");
+            assert(
+                new Date(accessTokenCookie.expires).toUTCString().endsWith("GMT"),
```

Comment:
toUTCString will always output in GMT I think. Please double check that this test fails with the old version.